SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchNavbar.tsx

```diff
@@ -330,65 +333,30 @@ const EventSearchNavbar = (props: IProps) => {
-                                                                A: value,
-                                                                B: value,
-                                                                C: value
-                                                            }
-                                                        });
-                                                    }} checked={(newEventCharacteristicFilter.Phase.A && newEventCharacteristicFilter.Phase.B && newEventCharacteristicFilter.Phase.C) ||
-                                                        !(newEventCharacteristicFilter.Phase.A || newEventCharacteristicFilter.Phase.B || newEventCharacteristicFilter.Phase.C)}
-                                                        disabled={!(newEventCharacteristicFilter.Phase.A || newEventCharacteristicFilter.Phase.B || newEventCharacteristicFilter.Phase.C)} />
-                                                <label className="form-check-label">Select All</label>
-                                                </div>
-                                            </div>
-                                            <div className="form-group">
-                                            <div className='form-check form-check-inline'>
-                                                <input className="form-check-input" type="checkbox" onChange={() => {
-                                                    setNewEventCharacteristicFilter({
-                                                        ...newEventCharacteristicFilter,
-                                                        Phase: {
-                                                            ...newEventCharacteristicFilter.Phase,
-                                                            A: !newEventCharacteristicFilter.Phase.A
-                                                        }
-                                                    });
-                                                }} checked={newEventCharacteristicFilter.Phase.A} />
-                                                <label className="form-check-label">A</label>
-                                            </div>
-                                            <div className='form-check form-check-inline'>
-                                                <input className="form-check-input" type="checkbox" onChange={() => {
-                                                    setNewEventCharacteristicFilter({
-                                                        ...newEventCharacteristicFilter,
-                                                        Phase: {
-                                                            ...newEventCharacteristicFilter.Phase,
-                                                            B: !newEventCharacteristicFilter.Phase.B
+                                            <MultiCheckBoxSelect
```

Comment:
This box seems super buggy. We might need a gemstone update to fix it before this gets merged. Select all button in particular is not super functional.

Reply:
Update: Issue was not gemstone but the way the state updates were setup. Bugs should be fixed